popup.js

```diff
@@ -0,0 +1,259 @@
+<li>${projects[6].technologies[2]}</li>
+</ul>
+<button class="button">See Project</button>
+</article>
+</section>
+`;
+
+document.body.insertBefore(articlesContainer, document.body.children[3]);
+
+const closebutton = document.querySelector('.popup #close');
+
+const button0 = document.querySelector('#multi .button');
+const button1 = document.querySelector('#one .button');
+const button2 = document.querySelector('#two .button');
+const button3 = document.querySelector('#three .button');
+const button4 = document.querySelector('#four .button');
+const button5 = document.querySelector('#five .button');
+const button6 = document.querySelector('#six .button');
+
+const projectWindow = document.querySelector('.popup');
+
+function showMenu0(a) {
+  a.preventDefault();
+  projectWindow.style.display = 'flex';
+  projectWindow.querySelector('h2').innerText = projects[0].title;
+  projectWindow.querySelector('.img_description #descriptioncontainer #description').innerText = projects[0].description;
+}
+
+function showMenu1(a) {
+  a.preventDefault();
+  projectWindow.style.display = 'flex';
+  projectWindow.querySelector('h2').innerText = projects[1].title;
+  projectWindow.querySelector('.img_description #descriptioncontainer #description').innerText = projects[1].description;
+  projectWindow.querySelector('#li4').remove();
+}
+
+function showMenu2(a) {
+  a.preventDefault();
+  projectWindow.style.display = 'flex';
+  projectWindow.querySelector('h2').innerText = projects[2].title;
+  projectWindow.querySelector('.img_description #descriptioncontainer #description').innerText = projects[2].description;
+  projectWindow.querySelector('#li4').remove();
+}
+
+function showMenu3(a) {
+  a.preventDefault();
+  projectWindow.style.display = 'flex';
+  projectWindow.querySelector('h2').innerText = projects[3].title;
+  projectWindow.querySelector('.img_description #descriptioncontainer #description').innerText = projects[3].description;
+  projectWindow.querySelector('#li4').remove();
+}
+
+function showMenu4(a) {
+  a.preventDefault();
+  projectWindow.style.display = 'flex';
+  projectWindow.querySelector('h2').innerText = projects[4].title;
+  projectWindow.querySelector('.img_description #descriptioncontainer #description').innerText = projects[4].description;
+  projectWindow.querySelector('#li4').remove();
+}
+
+function showMenu5(a) {
+  a.preventDefault();
+  projectWindow.style.display = 'flex';
+  projectWindow.querySelector('h2').innerText = projects[5].title;
+  projectWindow.querySelector('.img_description #descriptioncontainer #description').innerText = projects[5].description;
+  projectWindow.querySelector('#li4').remove();
+}
+
+function showMenu6(a) {
+  a.preventDefault();
+  projectWindow.style.display = 'flex';
+  projectWindow.querySelector('h2').innerText = projects[0].title;
+  projectWindow.querySelector('.img_description #descriptioncontainer #description').innerText = projects[6].description;
+}
```

Comment:
As I suggested in my previous comment, it would make your code a lot cleaner and more maintainable if you don't hard code each function for every project. Please be sure to simplify and avoid repeating code with the same functionality many times 🤓

Idea 💡 When you code your loop for creating your projects dynamically, you can add dynamic ids so you can use them later while checking for the clicked button and showing the appropriate info. 

https://www.w3schools.com/jsref/event_onclick.asp